crates/pipeline_manager/src/db/mod.rs

```diff
@@ -2255,6 +2279,176 @@ impl Storage for ProjectDB {
         let _res = conn.execute(&stmt, &[]).await?;
         Ok(())
     }
+
+    async fn new_external_service(
+        &self,
+        tenant_id: TenantId,
+        id: Uuid,
+        name: &str,
+        description: &str,
+        config: &ExternalServiceConfig,
+    ) -> Result<ExternalServiceId, DBError> {
+        debug!("new_external_service {name} {description} {config:?}");
+        let manager = self.pool.get().await?;
+        let stmt = manager
+            .prepare_cached("INSERT INTO connector (id, name, description, config, tenant_id) VALUES($1, $2, $3, $4, $5)")
```

Comment:
This looks wrong. It should not be inserting into the connector table.

Reply:
Fixed